src/dispatch/service/service.py

```diff
@@ -41,6 +42,7 @@ def create(*, db_session, service_in: ServiceCreate) -> Service:
         incident_types=incident_types,
         terms=terms,
     )
+    service.type = ONCALL_PLUGIN_SLUG
```

Comment:
Not sure if this is the only place where "pagerduty_oncall" is referenced, if you would like to help me with this @kevgliss would be awesome.

Reply:
afaik we only use the oncall_plugins in two places:

Here:
https://github.com/Netflix/dispatch/blob/develop/src/dispatch/incident/service.py#L42

and:
https://github.com/Netflix/dispatch/blob/develop/src/dispatch/incident/flows.py#L1088

Recently, we made a change that allows plugins to be configurable via the UI (instead of strictly via the config). I think in both places we should replace the _plugin.get_ calls with 
`plugin = plugin_service.get_active(db_session=db_session, plugin_type="service")` to enable this behavior (we're slowly migrating plugins as we add more of the same type).

Additionally, if you're up for it (this is specifically for services) we have on our backlog to be able to _specify_ the service's plugin (allowing you to mix and match services/plugins). Today we had just hardcoded pagerduty, but if we want folks to be able to specify ops-genie without knowing it's slug we should provide them a UI dropdown.

Reply:
Thank you for suggestions and hints, I've implemented the dropdown as you suggested